charts/seed-bootstrap/charts/etcd-druid/values.yaml

```diff
@@ -5,3 +5,11 @@ global:
 podAnnotations: {}
 labels:
   gardener.cloud/role: etcd-druid
+
+vpa:
+  enabled: true
+  minAllowed:
+    cpu: 50m
+    memory: 100M
+  updatePolicy:
+    updateMode: "Auto"
```

Comment:
Nice change, though, I'm wondering why you've put this into the `values.yaml` and not directly into the `vpa.yaml` directly? As we don't overwrite/configure it from the Go code (when rendering the chart), I would probably put it into the `vpa.yaml` directly.